hax/hax/queue/publish.py

```diff
@@ -45,6 +50,26 @@ def publish(self, message_type: str, payload: str) -> int:
             if ok:
                 return next_epoch
 
+    def publish_no_duplicate(self,
```

Comment:
That's a good catch!!

1. If an event is already published, we need to drop that **event**.  
2. For the epoch key being updated by other users.. then **we shouldn't drop that event.**

If `@repeat_if_fails` is added, then for case 1, we will do **infinite retries**. 

Maybe.. we can do something like this `@repeat_if_fails(max_retries=5)` for handling the case2. 
@trshaffer  @mssawant any better way?

Reply:
Even, I don't see any other purpose served by the `epoch_key` in this case where .. events pushed to EQ are in the format of `eq/<fid>`

@mssawant is it okay to avoid updating `epoch key` here?

Reply:
As per discussion with @mssawant, removing the updating of `epoch key`. Since the key format here for EQ is `eq/<fid>` which is uniquely identified.

Reply:
done